internal/remotes/fs.go

```diff
@@ -83,6 +84,27 @@ func (f *FileRemote) Push(id model.TMID, raw []byte) error {
 	return nil
 }
 
+func (f *FileRemote) Delete(id string) error {
+	err := f.checkRootValid()
+	if err != nil {
+		return err
+	}
+	err = checkIdValid(id)
+	if err != nil {
+		return err
+	}
+	match, _ := f.getExistingID(id)
+	if match != idMatchFull {
+		return ErrTmNotFound
+	}
+	fullFilename, _, _ := f.filenames(id)
+	err = os.Remove(fullFilename)
+	if os.IsNotExist(err) {
+		return ErrTmNotFound
+	}
+	return err
```

Comment:
Would it be possible to remove also parent folder structure up to the root, if it as the last TM ?
Or would that be too error-prone, e.g. checking if also TM's exists in sub folders (opt-tree / opt-path) ?

Reply:
added removing empty parent folders up to root. It seems to me, at this late hour on Friday evening, that there shouldn't be any problems with that :)